pkg/cli/email/validate.go

```diff
@@ -66,27 +70,36 @@ func Validate(emailAddress string, options ...Option) error {
 }
 
 func checkDisposableDomains(domain string) error {
+	logger := log.GetInstance()
+	if os.Getenv("DEBUG") == "true" {
+		logger.SetLevel(logrus.DebugLevel)
+	}
```

Comment:
I *think* a logger is already initialized with the correct level based on the `--debug` flag. Did `log.GetInstance()` not return a logger with that level already?

Reply:
I just removed it, since like you said there would be no work around anyway.